### PR TITLE
Add start tomcat server functionality. and fix issue no java debugger

### PR DESCRIPTION
**Pull request by adashen**
1. Add start tomcat server command. No war package is needed to start tomcat. And a root jsp will be created to show the deployed package link.
2. Fix the issue when java debugger is not installed. The issue happens when java debugger is not installed, and debug tomcat, the server will be suspended and cannot be stopped. Except kill process from command line. To fix the issue, instead of use suspend=y, change to n. And the server will not be hung any more.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -84,6 +89,10 @@
         }
       ],
       "view/item/context": [
+        {
+          "command": "tomcat.start",
+          "when": "view == tomcatServerExplorer && viewItem != runningserver"
```
Comment (Eskibear) [inline package.json]: Note `viewItem != runningserver` might be a potential issue if in the future items with other `contextValue` are added into this view.

File: src/Utility.ts [hunk 2]
```diff
@@ -130,8 +130,57 @@ export namespace Utility {
...
+
+    export async function writeFile(filePath: string, content: string): Promise<void> {
+        if (await fse.pathExists(filePath)) {
+            await fse.remove(filePath);
+        }
+        await fse.writeFile(filePath, content);
+    }
+
+    export async function getSubDirs(dir: string): Promise<string[]> {
+        const isDir: boolean = await isDirectory(dir);
```
Comment (jdneo) [inline src/Utility.ts]: ```isDir``` seems redundant here, since it is not used in other places of this method.

File: src/Utility.ts [hunk 3]
```diff
@@ -130,8 +130,57 @@ export namespace Utility {
         }
     }
 
+    export function generateRootPageString(packages: string[]): string {
```
Comment (Eskibear) [inline src/Utility.ts]: use `.jsp` instead of `.html` to dynamicly read dirs.

File: src/Tomcat/TomcatController.ts [hunk 4]
```diff
@@ -91,30 +98,42 @@ export class TomcatController {
...
+    }
+
     public async runOnServer(serverInfo: TomcatServer, packagePath: string, debug: boolean = false): Promise<void> {
+        await this.run(serverInfo, packagePath, debug);
+    }
+
+    public dispose(): void {
+        this.stopServers();
+        this._tomcat.saveServerListSync();
+        this._outputChannels.forEach((value: vscode.OutputChannel, key: string) => value.dispose());
```
Comment (jdneo) [inline src/Tomcat/TomcatController.ts]: redundant argument ```key``` here?

**Pull request merged**